lib/traxor.rb

```diff
@@ -21,4 +17,25 @@ def self.initialize_logger(log_target = STDOUT)
     end
     @logger
   end
+
+  def self.enabled?
+    @enabled ||= ENV.fetch('TRAXOR_ENABLED', true).present?
```

Comment:
I don't think this will return `false` as expected. For example, if an env var `TRAXOR_ENABLED=false` is set, then `ENV.fetch('TRAXOR_ENABLED', true)` will return the string `'false'` (not a boolean), and then `'false'.present?` returns `true`.
I think we want to ask for a positive on the env var. So if Traxor is disabled by default we would want
```ruby
@enabled ||= !ENV['TRAXOR_ENABLED'].nil?
```
Or if Traxor is enabled by default we would want
```ruby
@enabled ||= ENV['TRAXOR_DISABLED'].nil?
```
Let me know if I'm missing something here.